### ISSUE TITLE
World Creation Timeout

### ISSUE THREAD

**Issue by darkequine** [Sirttas/ElementalCraft#142]
**Describe the bug**
The world hangs on new creation. Can't be sure it's caused by your mod but it is listed in the latest.log file. I am using the most up-to-date versions of the mod and libraries.

**To Reproduce**

**Expected behavior**

**ModPack**
Custom Modpack, Eternium Expert
[latest.log](https://github.com/Sirttas/ElementalCraft/files/11401403/latest.log)

**Comment by Sirttas** [comment]
This happen in pretty standard code, could you try removing performances mods as they are known to break things.

**Comment by Sirttas** [comment]
Mainly modern fix as it adds the timeout itself (vanilla don't timeout, it waits.

**Comment by darkequine** [comment]
I removed ModernFix and Lightspeed and tried again. I am currently sitting at 30 minutes waiting and only at 1% world loading. Here is my modlist if it helps.
[modlist.txt](https://github.com/Sirttas/ElementalCraft/files/11401912/modlist.txt)
[latest.log](https://github.com/Sirttas/ElementalCraft/files/11401916/latest.log)

**Comment by darkequine** [comment]
Re-enabling ModernFix and Lightspeed, but downgrading Elemental Craft to 5.7.5 and DataPack Anvil to 4.3.2 allowed world creation to finish in about 12 minutes.

**Comment by Sirttas** [comment]
It may be caused by chipped: https://github.com/embeddedt/ModernFix/issues/73

**Comment by faeldray** [comment]
I came across this issue while trying to figure out why generating new chunks in my existing world (in my custom modpack) was causing everything to hang. I don't have ModernFix and Lightspeed in my pack but I do have Chipped (alongside Elementalcraft of course).

So I decided to do a little testing to see if Chipped was conflicting with Elementalcraft...but unfortunately it's not the problem. I made a small test pack with just these mods:

- dpanvil-1.19.2-4.4.0.jar
- elementalcraft-1.19.2-5.8.0.jar
- jei-1.19.2-forge-11.6.0.1015.jar
- Patchouli-1.19.2-77.jar

And the world creation keeps hanging. The latest one was been hanging at 54% for 15 minutes before I killed the task. Downgrading Elementalcraft and Datapack Anvil to the previous versions lets the world generate without issue. So it appears that the latest updates are the cause of the issue.

Edit: forgot to include the log: 
[2023-05-05-5.log](https://github.com/Sirttas/ElementalCraft/files/11404558/2023-05-05-5.log)
